vagrant/boxes.d/30-source.yaml

```diff
@@ -37,7 +37,8 @@ pulp2-nightly-pulp3-source-centos7:
     host_path: '..'
     guest_path: '/home/vagrant/devel'
     reverse: False
-  memory: 4096
+  memory: 12500
+  cpus: 4
```

Comment:
These look like your personal settings.

They are definitely not suitable for the average pulplift user.

Reply:
Expand the diff since it's misleading -- these are the settings for the box used for testing the migration plugin.  It's a lot more reasonable given that this box needs both Pulp 2 and Pulp 3 installed and running, plus is dealing with migrating potentially huge quantities of data, plus is doing this across multiple different simultaneous tasks.

Reply:
That makes more sense. Are you absolutely sure you want 12.5 GB though? That is very tight on a 16GB of RAM system, like I have. I personally will be unable to test it without shutting down my web browser. I will be strongly disinclined to fix bugs at 12.5GB.

Reply:
Would 10 be more reasonable for you?

Reply:
Yes, that would be feasible for my system.

Reply:
@mikedep333 Updated

Reply:
Thank you!